apps/src/code-studio/pd/application/facilitator1819/Facilitator1819Labels.jsx

```diff
@@ -165,4 +188,8 @@ const pageLabels = {
 const allLabels = Object.keys(pageLabels).reduce((allLabels, page) =>
   Object.assign(allLabels, pageLabels[page]), {});
 
-export {pageLabels, allLabels};
+const scoredQuestions = [
+  'resumeLink', 'csRelatedJobRequirements', 'diversityTrainingDescription', 'describePriorPd', 'additionalInfo'
+] + Object.keys(pageLabels['Section5YourApproachToLearningAndLeading']);
```

Comment:
Note, you can use the spread syntax here in the array definition `...Object.keys(pageLabels['Section5YourApproachToLearningAndLeading']);`